application/main.py

```diff
@@ -794,6 +794,31 @@ def get_model(fn):
         return response
     else:
         return send_file(path)
+    
+@application.route('/preprocess_bsdd/<id>', methods=['GET'])
+@login_required
+def preprocess_bsdd(user_data, id):
+    with database.Session() as session:
+        model = session.query(database.model).filter(
+            database.model.code == id).first()
+        bsdd_task = [task for task in model.tasks if task.task_type == "bsdd_validation_task"][0]
+        bsdd_results = [result.serialize() for result in bsdd_task.results]
+
+        if model.status_bsdd != 'n':
+            preprocessed_bsdd_data = {
+                'bSDD classification found': {
+                    'name': [r['classification_name'] for r in bsdd_results][0],
+                    'Release data': 'n.a.',
+                    'Organisation': 'BuildingSMART',
```

Comment:
Done https://github.com/Ghesselink/ifc-pipeline-validation/commit/b1610016bcf5115d7d43067c450df0fdc5e5d0ce

Reply:
> I think we should have default values for every field of the object in case something goes wrong/an information is not retrieved.

This is already the case.

    The default value of validity is invalid
    Default of 'domain_source' is 'classification_not_found https://github.com/Ghesselink/ifc-pipeline-validation/blob/19deaf9e934d8e6e79445b9bf86657cf68aeba32/application/bsdd_utils.py#L61
    All the counts defaults 0
    Classification_name defaults 'name not found' https://github.com/Ghesselink/ifc-pipeline-validation/blob/19deaf9e934d8e6e79445b9bf86657cf68aeba32/application/bsdd_utils.py#L86

Or do you think the default values should be different?